README.md

```
@@ -87,29 +90,61 @@ arius ls \
 ```
 
-### Account key
+### Updating
 
-Pass `-k` on the command line, set `ARIUS_KEY` environment variable, or store it in
-[.NET user secrets](https://learn.microsoft.com/aspnet/core/security/app-secrets):
+Run:
 
-```bash
-dotnet user-secrets set "arius:<account>:key" "<key>"
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)